src/routes/home.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
```

Comment:
From what I've seen, more standard express layout is to call these controllers, not routes - https://www.terlici.com/2014/08/25/best-practices-express-structure.html